_includes/manuals/nightly/2.1_quickstart_installation.md

```diff
@@ -100,6 +101,26 @@ echo "deb http://deb.theforeman.org/ plugins {{page.version}}" | sudo tee -a /et
 {% endhighlight %}
 </div>
 
+<div class="quickstart_os quickstart_os_debian12">
+  <p>
+    Enable Puppet's 8.x repository:
```

Comment:
I see the others use Puppet 7 and there are Puppet 7 packages for Bookworm. Should we first update everything to Puppet 8 or keep this consistent and use Puppet 7?

Reply:
I've seen the same and my brain went "well, we'll end up on p8 anyway, so lets use p8 and update the rest later (or before)"

I can submit a `s/7/8/` PR separately and then rebase this ontop?

Reply:
> I can submit a `s/7/8/` PR separately and then rebase this ontop?

I think that'd be great.

Reply:
https://github.com/theforeman/theforeman.org/pull/2187